Formula/git-filter-repo.rb

```diff
@@ -0,0 +1,35 @@
+class GitFilterRepo < Formula
+  desc "Quickly rewrite git repository history"
+  homepage "https://github.com/newren/git-filter-repo"
+  url "https://github.com/newren/git-filter-repo/releases/download/v2.24.0/git-filter-repo-2.24.0.tar.xz"
+  sha256 "92188d3c44b9ff0dd40dfeed72859e0a088f775c12fb24c4e3e27a8064cfcc84"
+
+  # ignore git dependency audit:
+  #  * Don't use git as a dependency (it's always available)
+  # https://github.com/Homebrew/homebrew-core/pull/46550#issuecomment-563229479
+  depends_on "git"
```

Comment:
Maybe specify a version to depend on?
```ruby
depends_on "git" => "2.22.0+"
```

Reply:
That syntax is invalid but it would be good to note in a comment, yeh.

Reply:
added with 0709214f4a